pom.xml

```diff
@@ -1750,6 +1750,7 @@
                                     -Xep:ObjectToString:ERROR
                                     -Xep:OptionalEquality:ERROR
                                     -Xep:OptionalMapUnusedValue:ERROR
+                                    -Xep:PreconditionsInvalidPlaceholder:ERROR <!-- Preconditions only accepts the %s placeholder in error message strings -->
```

Comment:
I don't think we need a comment here. It is easy to find it: https://errorprone.info/bugpattern/PreconditionsInvalidPlaceholder

Reply:
agreed